README.md

```diff
@@ -108,85 +111,64 @@ All commands listed below are of the Linux variety, and assume you already
 
-#### Creating a trading wallet for Bitcoin
+Unless you use Decrediton to start DEX, you will need a web browser to open the
+DEX client user interface as described in the next section.
 
-For Bitcoin, you can create a trading wallet using the bitcoin-cli utility.
-**bitcoin-cli** will be included in the quick-start installation as well. You
-can replace `dex` with whatever name you want. The rest of the instructions
-will assume you chose `dex`.
+### Optional External Software
 
-`./bitcoin-cli createwallet dex`
+Depending on which assets you wish to use, you have different choices for wallet
+software. There are native/built-in light wallet options for Bitcoin and Decred,
+an external light wallet option for Litecoin, and full-node support for all
+other assets including: Bitcoin, Decred, Litecoin, ZCash, Dogecoin, Bitcoin
+Cash. The following release will include Ethereum support with a native light
+wallet.
 
-It is recommended that you password-protect your Bitcoin trading wallet.
-We'll use `read` to prevent echoing the password.
+1. For Bitcoin, [Bitcoin Core v0.21 or v22 or v23](https://bitcoincore.org/en/download/) (bitcoind or bitcoin-qt) wallet. Descriptor wallets are not supported in v0.21. Alternatively, you man use the native light wallet.
+2. For Decred, [dcrd](https://github.com/decred/dcrd) and [dcrwallet](https://github.com/decred/dcrwallet), installed from the [v1.7.x release binaries](https://github.com/decred/decred-release/releases), or built from the `release-v1.7` branches. Alternatively you may use Decrediton or the dcrwallet application in SPV mode, or the native light wallet.
+3. For Litecoin, [Litecoin Core v0.21.x](https://litecoin.org/). Alternatively,
+you may use an external Electrum-LTC wallet with the name "default_wallet".
+4. For Dogecoin, [Dogecoin Core v1.14.5+](https://dogecoin.com/).
+5. For ZCash, [zcashd v5.1](https://z.cash/download/).
+6. For Bitcoin Cash, [Bitcoin Cash Node v24+](https://bitcoincashnode.org/en/)
 
-`read -s BTCPASS`
-
-Type your password and hit enter, then do
-
-`./bitcoin-cli -rpcwallet=dex encryptwallet $BTCPASS`
-
-You'll also want to instruct **bitcoind** to load the wallet at startup. Modify
-your **bitcoin.conf** file, located in the **~/.bitcoin** directory on Linux,
-**%APPDATA%\Bitcoin** on Windows, and **~/Library/Application Support/Bitcoin**
-on Mac OS. Open the file in a text editor and add the following line at the end
-of the file.
-
-`wallet=dex`
+See the [wiki](../../wiki/Testnet-Testing) for details on preparing the wallets.
 
 ### Connect Wallets and Register
 
-1. Start the client. `./dexc`
-2. In your browser, navigate to **localhost:5758**
+1. Start the client. `./dexc` if using DEX standalone, or the "DEX" tab within
+   Decrediton.
+2. In your browser, navigate to http://localhost:5758. Skip this step if
+   using Decrediton.
+
+[//]: # "TODO: either update or remove all of these screenshots for the current UI!"
```

Comment:
I'm leaning toward removing the screenshots (and most of the text in this section) because they can go out of date so quickly, and generally I'd like the README to be more terse, with this sort of setup walkthrough in the wiki.

Reply:
Keeping up with screenshots can be a pain.